spec/mailers/invoice_mailer_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe InvoiceMailer, type: :mailer do
+  describe 'notify_user_email' do
+    subject(:mail) { described_class.notify_user_email(user) }
+
+    let(:user) { create(:user) }
+
+    it 'renders the headers' do
+      expect(mail.subject).to eq('Comarev - Lembrete de expiração de fatura.')
+      expect(mail.to).to eq([user.email])
+      expect(mail.from).to eq(['test@test.com'])
```

Comment:
can you please add a spec to cover the body as well? It can be just for one work or a line

```rb
expect(mail.body).to match("Este é um lembrete de que sua fatura expira")
```
something like that 😄